terraform/modules/daos_server/scripts/daos_client_install_script.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Install DAOS Client package
+#
+# TODO: Add support for installing on openSUSE Leap 15.3 and Ubuntu 20.04 LTS
+#
+
+DAOS_VERSION="${DAOS_VERSION:-2.0}"
+
+echo "BEGIN: DAOS Client Installation"
+
+# Determine which repo to use
+# shellcheck disable=SC1091
+source "/etc/os-release"
+OS_VERSION=$(echo "${VERSION_ID}" | cut -d. -f1)
+OS_VERSION_ID="${ID,,}_${OS_VERSION}"
+case ${OS_VERSION_ID} in
+  centos_7)
+    DAOS_OS_VERSION="CentOS7"
+    ;;
+  centos_8)
+    DAOS_OS_VERSION="CentOS8"
+    ;;
+  rocky_8)
+    DAOS_OS_VERSION="CentOS8"
+    ;;
+  *)
+    echo "ERROR: Unsupported OS: ${OS_VERSION_ID}. Exiting."
+    exit 1
+    ;;
+esac
+
+echo "Adding DAOS v${DAOS_VERSION} packages repo"
+curl -s --output /etc/yum.repos.d/daos_packages.repo "https://packages.daos.io/v${DAOS_VERSION}/${DAOS_OS_VERSION}/packages/x86_64/daos_packages.repo"
+
+echo "Installing daos-client and daos-devel packages"
+yum install -y daos-client daos-devel
+systemctl enable daos_agent
```

Comment:
@markaolson   Is it safe to enable now or that should be with the installation of the YML files?  I was just working in the boot time area and there is a spot where the default YML gets uses and it is not great.

Reply:
It's only getting enabled at this point but it's not started yet. 
The service actually gets started after the yml files get deployed in a separate "configure" script.

Reply:
ok